userbot/modules/notes.py

```diff
@@ -64,7 +64,7 @@
         )
 
 
-@register(incoming=True, pattern=r"#\w*", disable_edited=True)
```

Comment:
```suggestion
@register(pattern=r"#\w*", disable_edited=True)
```
Actually we don't need incoming=True and outgoing=True together.

Reply:
Ah, sorry, I didnt know how the defaults worked if there were no args (literally just git cloned 15 mins ago). Will modify my fork!